config.dist.ini

```diff
@@ -0,0 +1,16 @@
+[firefox]
+windows=bin\webdrivers\gecko\geckodriver64.exe
```

Comment:
I don't think that we should go crazy with detection here. At the end you can always change config.dist.ini to config.ini and set the correct driver for your environment